ui/src/components/scroll-area/QScrollArea.js

```diff
@@ -317,8 +317,9 @@ export default createComponent({
       if (data.thumbHidden.value !== true) {
         const offset = evt[ dirProps[ axis ].offset ]
         if (offset < data.thumbStart.value || offset > data.thumbStart.value + data.thumbSize.value) {
-          const pos = offset - data.thumbSize.value / 2
-          setScroll(pos / container[ axis ].value * data.size.value, axis)
+          const targetThumbStart = offset - data.thumbSize.value / 2
+          const percentage = between(targetThumbStart / (props.container[ axis ].value - data.thumbSize.value), 0, 1)
```

Comment:
Hi,

There is no `container` prop. Did you mean `container` instead of `props.container`?

Reply:
Hi. You are correct! I have taken this from another branch that has `props.container`. New commit pushed.